08_files_lab/01.py

```diff
@@ -1,6 +1,12 @@
-"""
-Write a program that takes 2 file names
-and appends the second file's contents to
-the end of the first one
-"""
-
+import sys
+import os 
+
+(_, file1, file2) = sys.argv
+
+with open ("./" + file1, 'r') as fin:
+    with open ("./" + file2, 'w') as fout:  
+        for line in fin:
+            fout.write(line)
+            print line
+
+#should I delete file1 contents or just copy the lines to file2?
```

Comment:
Just copy is ok, but appending the provided filename to './' is not a good idea. What if the user gives you /tmp/a and /tmp/b ? Will it still work?